.travis.yml

```diff
@@ -3,7 +3,7 @@ language: node_js
 node_js:
   # we recommend testing addons with the same minimum supported node version as Ember CLI
   # so that your addon works for all apps
-  - "6"
+  - "10"
```

Comment:
Can you make this change in a separate PR? This should use Node 8 (not 10), as it is the minimum supported Node version. In addition, please make sure to also update the `package.json` `engines` field to show that Node 8 is the minimum supported version.

Reply:
Sure! That does make sense, I think at the time I was just trying to get things working, but this certainly makes sense

Reply:
Ya, I definitely understood that. I just like keeping them separate when we actually go to land the changes (makes reviewing the logs easier, changelog generation, etc).

Reply:
Added separate PR for updating CI configs https://github.com/nickiaconis/ember-prefetch/pull/151